tests/base.py

```diff
@@ -45,6 +46,12 @@
 from website.addons.base import AddonConfig
 from website.project.views.contributor import notify_added_contributor
 
+try:
+    DEFAULT_METASCHEMA = MetaSchema.find()[0]
+except IndexError:
+    ensure_schemas()
```

Comment:
Not a huge deal, but this should probably only execute within a test run. Right now, this is writing to the local database (rather than the test database). It might make sense to create a `get_default_metaschema()` function which is only ever executed in tests.